proposal.md

```diff
@@ -38,24 +38,36 @@ groceries index, cost index, purchasing index), the country’s economic
 status (GDP per capita), health benefits (life expectancy), social
 welfare (social support, freedom, generosity, corruption) and the
 happiness score of the country which provides information on how happy
-the population is. The data comes from two streams, the first stream
+the population is. The data comes from two streams: the first stream
 provides information about the aspects of happiness of a country and the
-second stream is about the statistics of living. Based on the client’s
-preference on the happiness aspects of a country, the visualization
+second stream is about the statistics of living. Based on clients'
+preference of various aspects of a country, the visualization
 provides comparison of the countries’ living conditions, which helps to
-narrow down the choices for the client for a country to live during
-retirement.
+narrow down the choices for the client to find the perfect country to live 
+during retirement.
 
 ## Research Questions and Usage Scenarios
 
+### Research Questions
+Some of the research questions we are hoping to answer for our customer 
+include:
+
+1. Rankings of countries in different region with respect to user 
+preferences such as social support and freedome to life choices. 
```

Comment:
small typo: freedom to make life choices

Reply:
fixed! thank you!